assets/styles/style.css

```diff
@@ -1,36 +1,109 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@200&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  font-family: Poppins, sans-serif;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-size: 1.5;
+  font-weight: 400;
+}
+
 .topic h1 {
-	font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
-	font-size: 30px;
-	margin-bottom: 20px;
+  font-size: 30px;
+  margin-bottom: 20px;
 }
 
 #background {
-	background-image: image("coder.jpg");
-	background-size: cover;
-	background-position: center;
+  background: linear-gradient(115deg, #3a3a9ecc, #8888ceb3), url("https://cdn.freecodecamp.org/testable-projects-fcc/images/survey-form-background.jpeg");
+  background-size: cover;
+  background-position: center;
+  background-attachment: fixed;
+  color: white;
 }
 
 .topic {
-	text-align: center;
+  text-align: center;
+  padding: 15px 0;
 }
+
 .topic p {
-	font-style: italic;
-	font-size: 20px;
-	margin-top: 10px;
+  font-style: italic;
+  font-size: 20px;
+  margin-top: 10px;
+  padding-bottom: 15px;
 }
 
 #informations {
-	padding-top: 10;
-	margin: 50px;
-	background-color: rgba(0, 0, 0, 50%);
-	
+  align-self: center;
+  padding-top: 25px;
+  padding-left: 30px;
+  margin-right: 350px;
+  margin-left: 350px;
+  padding-right: 200px;
+  border-radius: 10px;
+  background-color: #2c2b49;
+  opacity: 0.91;
+}
+
+#informations p {
+  padding: 10px 0;
+}
+
+.form {
+  width: 565px;
+  height: 30px;
+}
+
+#comments {
+  height: 100px;
+  width: 564px;
 }
 
 .question a {
-	padding-bottom: 10px;
+  padding-bottom: 10px;
+}
+
+.input-textarea {
+  font-size: 15px;
+  font-family: "Times New Roman", Times, serif;
+}
+
+.feature {
+  width: 611px;
+}
+
+.true {
+  margin: 10px 0;
+  line-height: 1.5 !important;
 }
 
 .name {
-	height: 30px;
-	width: 50%;
-}
\ No newline at end of file
+  height: 30px;
+  width: 560px;
+  margin: 15px 0;
+}
+
+.checkbox {
+  line-height: 1.5 !important;
+}
+
+.submit {
+  padding: 20px 0 50px;
+}
+
+.button {
+  width: 570px;
+  display: block;
+  cursor: pointer;
+  color: #fff;
+  border-radius: 3px;
+  font-size: 20px;
+  padding: 0.75rem;
+  background-color: #37af65;
+  border: none;
+}
```

Comment:
I am interested that you are making efforts to improve your project. That's good

![Screenshot from 2023-04-05 21-52-24](https://user-images.githubusercontent.com/116273730/230208974-2b9cfe60-d66e-45e1-b18c-1a2f49086f24.png)